packages/space-header/src/inject_to_body.ts

```diff
@@ -0,0 +1,11 @@
+export const inject_to_body = (
```

Comment:
perhaps we should add a class name to make it easier for integrators to change the position?

that way they don't have to change their underlying app/page design in case the header gets in the way of something (toolbar etc)

Reply:
Ahh got it! 

I missed the `box.setAttribute("id", "huggingface-space-header");`

all good 👍

Reply:
Related: I think we should allow users to provide a "target" and if so, rather than appending to `body`, append to that target.

Like with Svelte: (but same for any other framework)

https://svelte.dev/docs/client-side-component-api#creating-a-component

eg `init(space, {target: "..."})`

Reply:
@coyotte508 should be good now! tell me if you had something different in mind 🤗

Reply:
In my mind it was more of a string that can be passed to `document.querySelector`, but a `HTMLElement` works too